### PR TITLE
test/137/write-test-for-Crads-list

### LINKED ISSUE #137: Write test for CartList

### PR DESCRIPTION
**Pull request by RomanKhorol**
I've created new issue and done Crads-list test .

### CODE REVIEW

**Review by CallMeMaverick — COMMENTED** [review]
I assume that it would be a good practice to add a description for each test in the suite, to provide a context

**Comment by RomanKhorol** [comment]
> I assume that it would be a good practice to add a description for each test in the suite, to provide a context

**Comment by RomanKhorol** [comment]
> > I assume that it would be a good practice to add a description for each test in the suite, to provide a context

Yes of course. It was my mistake.

**Review by kryzanivska-nastya — APPROVED** [review]

**Review by Bakalina — APPROVED** [review]

**Review by CallMeMaverick — APPROVED** [review]

File: src/tests/unit/components/cards-list/CardsList.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+describe('Card List', () => {
+  let loading = true
+  let cards = []
+  test('renders Loader when loading and no cards', () => {
+    render(<CardsList cards={cards} loading={loading} />)
+    const loader = screen.getAllByTestId('loader')[0]
+    expect(loader).toBeInTheDocument()
+  })
+  test('shows card when cards length is', () => {
```
Comment (nrevutska) [inline src/tests/unit/components/cards-list/CardsList.spec.jsx]: add what length

File: src/tests/unit/components/cards-list/CardsList.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    let cards = ['test']
+    render(<CardsList cards={cards} loading={loading} />)
+    const card = screen.getByText('test')
+    expect(card).toBeInTheDocument()
```
Comment (nrevutska) [inline src/tests/unit/components/cards-list/CardsList.spec.jsx:18]: is this the correct behavior given that loading is true?

**Review by nrevutska — CHANGES_REQUESTED** [review]
Fix what is needed and then you can merge pull request without additional reviews

File: src/tests/unit/components/cards-list/CardsList.spec.jsx (hunk 2 above)
Comment (RomanKhorol) [reply src/tests/unit/components/cards-list/CardsList.spec.jsx:18]: yes

File: src/tests/unit/components/cards-list/CardsList.spec.jsx (hunk 1 above)
Comment (RomanKhorol) [reply src/tests/unit/components/cards-list/CardsList.spec.jsx]: Ok

**Pull request merged**